Controllers/Customer_UsersController.cs

```diff
@@ -181,7 +164,7 @@ public async Task<IActionResult> Edit(int? id)
 
             ViewBag.Customers = appUserCustomers;
 
-            var appUsers = _recitopiaDbContext.AppUsers;
+            var appUsers = await _recitopiaDbContext.AppUsers.ToListAsync();
 
             IEnumerable<SelectListItem> appUserList = _recitopiaDbContext.AppUsers.Select(c => new SelectListItem
```

Comment:
This is only a query expression until you call ToList(), ToListAsync(), etc.

Entity Framework has delayed execution until you call a method to retrieve the data.

Reply:
removed unused call